src/main/java/seedu/address/commons/util/PrefixUtil.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.commons.util;
```

Comment:
Im fine with this, but what about defining an abstract method on EntityList and Entity called `getPrefixType`

Reply:
I can also do that